testcontainers/src/clients/http.rs

```diff
@@ -416,7 +391,7 @@ mod tests {
         let image = RunnableImage::from(image).with_container_name("hello_container");
         let container = docker.run(image).await;
 
-        let container_details = inspect(&docker.inner.bollard, container.id()).await;
+        let container_details = inspect(&docker.bollard, container.id()).await;
```

Comment:
Can you rewrite these tests to use the new API so we get an idea what it feels like / whether it works?

Reply:
I just pushed a commit with updated test. Let me know how it looks.

Reply:
Looks good!